connector_node/connector-api/src/main/java/com/risingwave/connector/api/sink/SinkFactory.java

```diff
@@ -5,4 +5,6 @@
 
 public interface SinkFactory {
     SinkBase create(TableSchema tableSchema, Map<String, String> tableProperties);
+
+    void validate(TableSchema tableSchema, Map<String, String> tableProperties);
```

Comment:
expect to change Map into some struct. Maybe later 😇

Reply:
I think a map is more flexible, because different sinks may define their own properties.